corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -56,7 +56,7 @@ def get_commcare_users_by_filters(domain, user_filters, count_only=False):
                 user_filters.get('location_id', None), count_only]):
         return get_all_commcare_users_by_domain(domain)
 
-    query = _get_es_query(domain, user_filters)
+    query = _get_es_query(domain, user_filters).remove_default_filter('active')
```

Comment:
confirming that you have seen all the usages of `get_commcare_users_by_filters`?
Also should we now rename it to `get_active_commcare_users_by_filters`?

Reply:
Yep yep. I have. It is only used in `bulk_download.py` and `users.py` which are serving this use-case only.
We are actually returning all the users, we have removed the default active filter.

Reply:
So this change will effect only the count we fetch while downloading users?

Reply:
As I looked into the functioning we pass in `count_only` as True whenever we have to get count and that uses ES else to get user `get_all_commcare_users_by_domain` is called. So I guess it would be a yes.